lib/federated/generator.rb

```diff
@@ -12,11 +12,21 @@ def create!(options={})
       relayable = build(options)
       if relayable.save!
         logger.info "user:#{@user.id} dispatching #{relayable.class}:#{relayable.guid}"
+        add_root_author(relayable)
         Postzord::Dispatcher.defer_build_and_post(@user, relayable, @dispatcher_opts)
         relayable
       end
     end
 
+    def add_root_author(relayable)
+      # If comment post is a reshare, include original author in subscribers
+      if relayable.parent.respond_to?(:root) && relayable.parent.root
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.